libs/blocks/quiz/quiz.css

```diff
@@ -283,36 +283,27 @@
 }
 
 /* Quiz button */
-.quiz-button-container .quiz-btn {
-  padding: 0.5938rem 5.3438rem;
-  border-color: var(--color-white);
-  background: transparent;
-  border-radius: 1.875rem;
-  border-width: 0.125rem;
-  box-sizing: border-box;
-  height: 3rem;
-}
-
 .quiz-button-container .quiz-btn {
   border-style: solid;
   font-size: 14px;
   font-weight: 700;
   min-height: 32px;
   min-width: 72px;
-  padding: 0.5938rem 5.3438rem;
+  padding: 0.5938rem 4.3438rem;
   border-color: var(--color-white);
   background: transparent;
   border-radius: 1.875rem;
   border-width: 0.125rem;
   box-sizing: border-box;
   height: 3rem;
```

Comment:
I've no idea why your branch won't render. After you manually merged changes and it still didn't work, I updated uar-integration from main and then updated your branch from the git ui. Still no joy. 

In any case, though, I just pulled your PR locally and tried updating button text using silly values in the inspector. 

I think we still run into problems because the button height is set to a fixed value of 3 rem. 

<img width="1288" alt="Screenshot 2023-08-24 at 11 22 22 AM" src="https://github.com/adobecom/milo/assets/25230829/8e3b7d16-df8e-4a0b-a56a-560d42117b79">

Setting min-height, rather than height seems to make things work as I hack around.

Reply:
To test locally, I just used: gh pr checkout 1153